packages/nimble-components/src/anchor-tree-item/index.ts

```diff
@@ -0,0 +1,136 @@
+export class AnchorTreeItem extends AnchorBase {
+    /**
+     * When true, the control will appear selected by user interaction.
+     * @public
+     * @remarks
+     * HTML Attribute: selected
+     */
+    @attr({ mode: 'boolean' })
+    public selected = false;
+
+    /**
+     * When true, the control will be immutable by user interaction. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled | disabled HTML attribute} for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: disabled
+     */
+    @attr({ mode: 'boolean' })
+    public disabled = false;
+
+    /**
+     * Whether the tree is nested
+     *
+     * @public
+     */
+    public isNestedItem(): boolean {
+        return isTreeItemElement(this.parentElement as Element);
+    }
+
+    /**
+     * Handle focus events
+     *
+     * @internal
+     */
+    public handleFocus(_e: FocusEvent): void {
+        this.setAttribute('tabindex', '0');
+    }
+
+    /**
+     * Handle blur events
+     *
+     * @internal
+     */
+    public handleBlur(_e: FocusEvent): void {
+        this.setAttribute('tabindex', '-1');
+    }
+
+    /**
+     * @internal
+     */
+    public keydownHandler(e: KeyboardEvent): boolean {
+        if (e.defaultPrevented) {
+            return false;
+        }
+        switch (e.key) {
+            case keyEnter:
+                // Do not let the event bubble up to the FAST tree, or it will
+                // prevent the default action.
+                e.stopPropagation();
+                break;
+            case keyArrowLeft:
+                // For FAST tree items, the FAST tree view handles this navigation,
+                // but since our anchor tree item is not "instanceof FASTTreeItem",
+                // the FAST tree view won't do this for us. We do it ourselves.
+                if (this.parentElement && this.isNestedItem()) {
+                    FoundationTreeItem.focusItem(this.parentElement);
+                }
+                break;
+            default:
+        }
+        return true;
+    }
+
+    /**
+     * Apparently, pressing the Enter key on an anchor generates a click event, too.
+     * This bubbles up to the Nimble tree-view's click handler, causing the tree item
+     * to be selected. We don't want that for anchor tree items. We'll stop propagation
+     * of the event to prevent that.
+     * @internal
+     */
+    public clickHandler(e: MouseEvent): boolean {
+        if (e.defaultPrevented) {
+            return false;
+        }
+        e.stopPropagation();
+        return true;
+    }
+
+    protected selectedChanged(
+        _prev: boolean | undefined,
+        _next: boolean
+    ): void {
+        if (this.$fastController.isConnected) {
+            this.$emit('selected-change', this);
+        }
+    }
+}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface AnchorTreeItem extends StartEnd {}
+applyMixins(AnchorTreeItem, StartEnd);
```

Comment:
So it looks like the Foundation Anchor base class is [already mixing in StartEnd slots](https://github.com/microsoft/fast/blob/c5d83a19ffdeb1d0a9680385ae907b342716b4e5/packages/web-components/fast-foundation/src/anchor/anchor.ts#L177-L178). I think I got confused as some PRs switched back and forth on extending the AnchorBase class.

Can you make a pass to confirm that all the Anchors that extend AnchorBase do not apply the StartEnd mixin a second time and that everything extending anchor base has a start and end slot in their template?

So in this file we do not need to mixin the start and end slot. May be worth leaving a comment saying that it is mixed in by the foundation class or something.

Reply:
Found a couple that were applying the mixin a second time, and found one that was not including the start/end slots in their template. Fixed now.

I also added comments about the mixin already being applied by FoundationAnchor.